services/datamanager/builtin/file_deletion.go

```diff
@@ -34,11 +34,14 @@ func shouldDeleteBasedOnDiskUsage(ctx context.Context, captureDirPath string, lo
 		return false, nil
 	}
 	if usedSpace < fsThresholdToTriggerDeletion {
-		logger.Warnf("disk not full enough, exiting used space: %f", usedSpace)
```

Comment:
Think this can still be helpful to have in a debug log with usage.Available() and usage.Size() printed, so we can remotely help a user troubleshooting this.